src/pkg/backup/details/details.go

```diff
@@ -1044,3 +1044,8 @@ func updateFolderWithinDrive(
 
 	return nil
 }
+
+// ExtensionInfo describes extension data associated with an item
+type ExtensionInfo struct {
+	Data map[string]any `json:"data,omitempty"`
```

Comment:
Is the plan to add an `ExtensionInfo` field to the core item info structs? Doesn't need to be done in this PR but would like to understand exactly how this struct is going to be linked to the item info (e.g. OneDrive)

Reply:
yep! I'll link `ExtensionInfo` to item info in a follow up PR. Trying to keep PRs short.